### PR TITLE
Add responsivness to topbar and banner

### PR DESCRIPTION
**Pull request by lkulbacki**
Due to column stacking and reordering requirement I decided to change banner HTML structure in lower resolutions - therefore two "company-claim" elements are visible in index.html

https://projects.kodilla.com/browse/WDP181201-12

### CODE REVIEW

**Comment by pawel-schmidt** [comment]
Resolve conflicts.

File: src/sass/components/_header.scss [hunk 1]
```diff
@@ -238,3 +258,58 @@ header {
...
+
+    .company-claim-md {
+      display: block;
+
+      .container > .row {
+        padding-top: 0;
+      }
+
+      .phone-number {
+        text-align: left !important;
```
Comment (stepek) [inline src/sass/components/_header.scss]: Please don't use `! important`

**Comment by lkulbacki** [comment]
Comments fixed, conflict resolved

File: src/index.html [hunk 2]
```diff
@@ -14,17 +14,17 @@
   <div class="top-bar">
     <div class="container">
       <div class="row">
-        <div class="col text-left">
+          <div class="col-8 col-md-6 text-left">
```
Comment (pawel-schmidt) [inline src/index.html]: Incorrectly indented line. Lines below also.

Comment (lkulbacki) [reply src/index.html]: Fixed

**Pull request merged**